packages/mutation-testing-elements/src/components/mutation-test-report-totals/index.ts

```diff
@@ -128,11 +130,18 @@ export class MutationTestReportTotalsComponent extends LitElement {
     </tr>`;
   }
 
-  private determineColoringClass(score: number) {
-    if (this.thresholds) {
-      if (score < this.thresholds.low) {
+  private mutationScoreRounded(mutationScore: number) {
+    if (!isNaN(mutationScore)) {
```

Comment:
This if is not needed anymore:

```sh
> 3.3.toFixed(2)
'3.30'
> NaN.toFixed(2)
'NaN'
```

Reply:
And then later I should check for the string 'NaN'?